packages/ui-components/src/components/breadcrumb-list/test/breadcrumb-list.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { newSpecPage, SpecPage } from '@stencil/core/testing';
+import { KvBreadcrumbList } from '../breadcrumb-list';
+import { h } from '@stencil/core'
+
+describe('KvBreadcrumbItem (unit tests)', () => {
+	let page: SpecPage;
+	let component: KvBreadcrumbList;
+
+	describe('when the component loads', () => {
+		describe('and doesnt have a separator', () => {
+			beforeEach(async () => {
+				page = await newSpecPage({
+					components: [KvBreadcrumbList],
+					template: () => <kv-breadcrumb-list></kv-breadcrumb-list>,
```

Comment:
Tests should be done on plain HTML, if it works with that it works with everything.

```suggestion
					html: `<kv-breadcrumb-list></kv-breadcrumb-list>`,
```